lib/searchkick/query.rb

```diff
@@ -464,6 +469,10 @@ def prepare
         payload[:explain] = options[:explain] if options[:explain]
         payload[:profile] = options[:profile] if options[:profile]
 
+        # scroll
+        raise ArgumentError, "From parameter cannot be used with scroll" if scroll && !payload[:from].nil?
```

Comment:
Don't think we need to check for this case - can just let Elasticsearch return an error